blog/2023-09-15-teams-feature-overview/index.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Make the dream work! An overview of Stately Studio's teams feature
```

Comment:
Best title ever! Awesome.

Reply:
haha thanks! Had to sprinkle a bit of cheese on the post.